packages/swirl-components/src/components/swirl-table/swirl-table.tsx

```diff
@@ -99,26 +101,32 @@ export class SwirlTable {
     const columns = this.getColumns();
 
     columns.forEach((column) => {
-      column.classList.remove("table-column--has-shadow");
+      column.classList.remove(
+        "table-column--has-shadow",
```

Comment:
**nitpick:** as this is the same list as below it might be a variable that holds this information